sdk/storage/azure-storage-file-datalake/tests/test_large_file.py

```diff
@@ -74,14 +75,17 @@ def test_append_large_stream_without_network(self, datalake_storage_account_name
         # Act
         response = file_client.append_data(data, 0, LARGEST_BLOCK_SIZE)
 
-        self.assertIsNotNone(response)
-        self.assertEqual(self.payload_dropping_policy.append_counter, 1)
-        self.assertEqual(self.payload_dropping_policy.append_sizes[0], LARGEST_BLOCK_SIZE)
+        assert response is not None
+        assert self.payload_dropping_policy.append_counter == 1
+        assert self.payload_dropping_policy.append_sizes[0] == LARGEST_BLOCK_SIZE
 
     @pytest.mark.skip(reason="Pypy3 on Linux failed somehow, skip for now to investigate")
```

Comment:
Can we make this Skip Pypy specific? I believe there is a way to do this with `@pytest.mark.skipif` and some form of version check. The version check may be similar to the PyPy check in the datalake conftest.

Reply:
Ooo. I will give that a try!